tools/gen-intrinsics-specs.py

```diff
@@ -544,6 +578,128 @@ def recurse_print_to_rst(item, section_level_list, headers=__intrinsic_table_hea
     ===========  ======================  =====================  ========  =========================
               1                       2                      3         4                          5
     ===========  ======================  =====================  ========  =========================
+    >>> table_item = ('__intrinsic_table', [[1,2,3,4,5], [6,7,8,9, 10]])
```

Comment:
Nit: instead of resetting `table_item` or `item` for each table, how about interleaving each call of `recurse_print_to_rst`?

Something like:

```
>>> table_item = ('__intrinsic_table', [[1,2,3,4,5], [6,7,8,9, 10]])
>>> print(recurse_print_to_rst(table_item, ['=']))
... expected output
>>> print(recurse_print_to_rst(table_item, ['#'], tablefmt='github'))
... expected output 
>>> item = ....
>>> print(recurse_print_to_rst(item, ['=']))
.... expected grid output
>>> print(recurse_print_to_rst(item, ['#'], tablefmt='github'))
... expected github output
```

This would make the tests easier to handle because they will be based on a single set of input values. Change the values, both tests need to be changed.